src/kernel/chainparams.cpp

```diff
@@ -246,6 +247,8 @@ class CTestNetParams : public CChainParams {
         vSeeds.emplace_back("seed.tbtc.petertodd.net.");
         vSeeds.emplace_back("seed.testnet.bitcoin.sprovoost.nl.");
         vSeeds.emplace_back("testnet-seed.bluematt.me."); // Just a static list of stable node(s), only supports x9
+        vSeeds.emplace_back("dnsseed.testnet.bitcoin.achow101.com"); // Ava Chow, only supports x1, x5, x9, x49, x809, x849, xd, x400, x404, x408, x448, xc08, xc48, x40c
```

Comment:
Looks like it's missing "." at the end (all the other seeder DNS names have this)--same for the testnet one

Reply:
Done